client/src/app/pages/applications/application-assessment/components/assessment-stakeholders-form/assessment-stakeholders-form.tsx

```diff
@@ -39,11 +39,10 @@ export const AssessmentStakeholdersForm: React.FC = () => {
   const { setValue, control, formState } =
     useFormContext<ApplicationAssessmentWizardValues>();
 
-  const { stakeholders } = useFetchStakeholders();
+  // const { stakeholders } = useFetchStakeholders();
```

Comment:
Note to self - stakeholder work is on the TODO list

Reply:
Yeah we need the hub guys to add stakeholders/stakeholderGroups to the assessment struct.